README.md

```diff
@@ -27,6 +31,54 @@ It can be used in one of the following ways.
 - Use `-debug` of `--debug` option
 - Set `SD_CMD_DEBUG_LOG` environment variable to `true`
 
+### Publish
+We publish the command specified by a yaml.
+```bash
+USAGE:
+   sd-cmd publish [command options]
+
+OPTIONS:
+   -f, --f    Specify the path of yaml to publish (default: sd-command.yaml)
+   -t, --t    Specify the tag given to the command (default: sd-command.yaml)
+
+EXAMPLE:
+   sd-cmd publish -f ./sd-command.yaml -t latest
+```
+
+### Validate
+We validate that a yaml is in the correct format for the command.
```

Comment:
```suggestion
Validating if a yaml is correct.
```

Reply:
[IMO] This seems the `validate` only checks the yaml is in correct format as yaml.

Reply:
Is it better?
Validating if yaml is correct as sd-cmd format.

Reply:
It looks better for me.